Yesod/Auth/OAuth2.hs

```diff
@@ -57,9 +58,22 @@ authOAuth2 :: YesodAuth m
            --   second authorized request to @api/me.json@.
            --
            --   See @'fromProfileURL'@ for an example.
-           --
            -> AuthPlugin m
-authOAuth2 name oauth getCreds = AuthPlugin name dispatch login
+authOAuth2 name oauth getCreds = authOAuth2Widget name oauth getCreds Nothing
```

Comment:
Instead of using a `Maybe`, how about we just pass `[whamlet|Login via #{name}|]` here?